resources/logging/charts/fluent-bit/values.yaml

```diff
@@ -9,7 +9,7 @@ replicaCount: 1
 image:
   repository: eu.gcr.io/kyma-project/tpi/fluent-bit
   pullPolicy: IfNotPresent
-  tag: 1.5.7-a558c5c4
+  tag: 1.8.4-PR-117
```

Comment:
can you please change this to proper image version since PR is merged already in tpi repo

Reply:
done